### PR TITLE
Added new example showing how to move a participant between calls.

### PR DESCRIPTION
**Pull request by navali-msft**

### CODE REVIEW

File: api-reference/beta/api/participant-invite.md [hunk 1]
```diff
@@ -917,6 +917,187 @@ Content-Type: application/json
 
 >**Note:** With a `completed` status, you can expect to receive notifications about how your original peer-to-peer call has been terminated and deleted.
 
+### Example 5: Moving a participant to a different meeting
+
+A participant can be moved from one meeting to another meeting, given that these two meetings are created by the same application.
+For more information about how to create an online meetings, see [create onlineMeeting](/graph/application-post-onlinemeetings).
+
+> **Note:** The invite API supports only one participant when moving between calls.
```
Comment (JarbasHorst) [inline api-reference/beta/api/participant-invite.md:925]: For simplicity and clarity, I recommend the following revision:

- **Note:** The invite API can move only one participate per request. If you provide more than one participant in the request body, the invite API will move only the first one.


Same here: api-reference/v1.0/api/participant-invite.md

File: api-reference/beta/api/participant-invite.md [hunk 2]
```diff
@@ -917,6 +917,187 @@ Content-Type: application/json
 
 >**Note:** With a `completed` status, you can expect to receive notifications about how your original peer-to-peer call has been terminated and deleted.
 
+### Example 5: Moving a participant to a different meeting
+
+A participant can be moved from one meeting to another meeting, given that these two meetings are created by the same application.
+For more information about how to create an online meetings, see [create onlineMeeting](/graph/application-post-onlinemeetings).
```
Comment (JarbasHorst) [inline api-reference/beta/api/participant-invite.md:923]: Please, make sure that the reference link to **create onlineMeeting** is correct. 

Same here: api-reference/v1.0/api/participant-invite.md

File: api-reference/beta/resources/invitationparticipantinfo.md [hunk 3]
```diff
@@ -21,7 +21,8 @@ Represents an entity that is being invited to a group call.
 | :--------------------------------- | :---------------------------- | :----------------------------------------------------------------------------------- |
 | endpointType                       | String                        | The type of endpoint. Possible values are: `default`, `voicemail`. |
 | identity                           | [identitySet](identityset.md) | The [identitySet](identityset.md) associated with this invitation.                   |
-| replacesCallId                     | String                        | Optional. The call which the target idenity is currently a part of. This call will be dropped once the participant is added. |
+| replacesCallId                     | String                        | Optional. The call which the target idenity is currently a part of. For peer-to-peer case, the call will be dropped once the participant is added successfully. |
+| participantId                      | String                        | Optional. The id of the target participant.                                          |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/invitationparticipantinfo.md:25]: Please, include an update to the changelog file to surface this new property (v1.0 and beta) in the Microsoft Graph Changelog.

**Comment by JarbasHorst** [comment]
Hello @navali-msft, I reviewed your PR and left some comments. Please, take a look when you will have time. 

Also, to avoid merge conflicts, please merge PR [16633](https://github.com/microsoftgraph/microsoft-graph-docs/pull/16633) into your branch to include my fixes and before addressing my comments.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [16675](https://github.com/microsoftgraph/microsoft-graph-docs/pull/16675) into this branch to include my final edits.

@navali-msft, please don't readd the label "ready for content review" when the content review is already in progress for a PR. My team uses the label "ready for content review" as an indication that the content review work should be planned for a new PR, which in this case might cause confusion.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**